src/test/java/org/synyx/urlaubsverwaltung/ui/CustomAbsenceTypeUIIT.java

```diff
@@ -95,6 +95,7 @@ void ensureAddingAndEnablingCustomVacationType(Page page) {
         navigationPage.clickSettings();
         settingsPage.clickAbsenceTypesTab();
         settingsPage.addNewVacationType();
+        page.waitForTimeout(2000);
```

Comment:
es gibt ein [waitForResponse](https://playwright.dev/java/docs/api/class-page#page-wait-for-response). vielleicht ist das elegant zu nutzen innerhalb von `addNewVacationType`

Reply:
ja, ich bin gerade am Suchen. Genau sowas brauchen wir. Ich teste mal.